recipes/pygslodeiv2/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pygslodeiv2" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "808c11692eab55ab16b99ebcc2e41b3eab1e8d303e1d70c208ed143da62310f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx and py27]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - pip
+    - numpy 1.11.*
+    - gsl
```

Comment:
Are you linking to openblas with gsl?

Reply:
Indirectly?:
https://github.com/conda-forge/gsl-feedstock/blob/master/recipe/meta.yaml#L36
https://github.com/bjodah/pygslodeiv2/blob/master/pygslodeiv2/_config.py#L2
https://github.com/bjodah/pygslodeiv2/blob/master/setup.py#L59
so I link against `-lgslcblas`, I could change that by setting `PYGSLODEIV2_BLAS=openblas`.
(sorry, I'm a bit fuzzy on exactly what linkers do when linking against a shared library which was linked against another one...)

Reply:
In conda-forge gslcblas is a link to openblas. So you need to pin openblas here. It's better to add `PYGSLODEIV2_BLAS=openblas` as well.